src/ontology/fbbt-edit.obo

```diff
@@ -122569,6 +122585,21 @@ relationship: part_of FBbt:00110893 ! adult midgut region R2
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-06-07T11:33:30Z
 
+[Term]
+id: FBbt:00058222
+name: sugar-sensing neuron of the adult protocerebrum
+def: "Chemosensory neuron in the posterior superior lateral protocerebrum that senses sugars in the hemolymph to regulate feeding. There are 4 of these cells per brain hemisphere. They express the fructose receptor Gr43a and the sugar receptor Gr64a, and secrete corazonin (Miyamato and Amrein, 2014; Fuji et al., 2015)." [FlyBase:FBrf0224445, FlyBase:FBrf0227698]
+subset: cur
+intersection_of: FBbt:00005106 ! neuron
+intersection_of: expresses http://flybase.org/reports/FBgn0041243 ! Gr43a
+intersection_of: has_soma_location FBbt:00007145 ! adult protocerebrum
```

Comment:
if these are the only Gr43a cells of the brain, you could widen this to 'adult brain'

Reply:
Yes, they are the only Gr43a-expressing neurons in the brain.

So with the comment below, you mean (1) logically define as "part of some adult brain" (and change name accordingly) and (2) with a "has soma location" relationship to give the precise location?

Reply:
I generally like to grab as much as possible (as long as it is correct) with the logical def, then add further detail as SubClassOf annotations, rather than making the logical definition overly specific.

I think the name is probably fine, unless another sugar-sensing cell type is identified in the protocerebrum - then we would need to rename, but I think they also looked at some other sugar receptors(?) so maybe not likely that there would be another unidentified type. Maybe putting 'Gr43a' in the name would be more accurate than 'sugar-sensing'? - up to you whether you want to change.